workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/pages/TicketApp.tsx

```diff
@@ -81,110 +80,92 @@ const App: React.FC<AppProps> = ({queryClient}) => {
-					<ul>
-						<li>
-							<a href="/">Dashboards</a>
-						</li>
-						<li>
-							<a href="/">Projects</a>
-						</li>
-						<li>
-							<a href="/">Issues</a>
-						</li>
-					</ul>
-				</nav>
-			</div>
-			<div className="col-md-10">
-				<header className="align-items-center bg-light mb-3 p-3 row">
-					<h1>Your Tickets</h1>
-					<button
-						className="btn btn-primary ml-auto"
-						onClick={(event) => {
-							mutation.mutate();
-
-							event.preventDefault();
+		<>
+			<header className="align-items-center bg-light mb-3 p-3 row">
+				<h1>Your Tickets</h1>
+				<button
+					className="btn btn-primary ml-auto"
+					onClick={(event) => {
+						mutation.mutate();
+
+						event.preventDefault();
+					}}
+				>
+					Generate a New Ticket
+				</button>
+			</header>
+
+			<main className="p-0 row">
+				<div className="col-md-10 m-0 p-0 pr-3">
+					<input
+						className="form-control mb-3 w-100"
+						onChange={(event) => {
+							setSearch(event.target.value);
+							setPage(1);
 						}}
-					>
-						Generate a New Ticket
-					</button>
-				</header>
-
-				<main className="p-0 row">
-					<div className="col-md-10 m-0 p-0 pr-3">
-						<input
-							className="form-control mb-3 w-100"
-							onChange={(event) => {
-								setSearch(event.target.value);
-								setPage(1);
+						placeholder="Search Tickets"
+						type="text"
+					></input>
+
+					<TicketGrid tickets={tickets} />
+
+					<div className="my-3">
+						<ClayPaginationBarWithBasicItems
+							active={page}
+							activeDelta={pageSize}
+							ellipsisBuffer={3}
+							ellipsisProps={{
+								'aria-label': 'More',
+								'title': 'More',
 							}}
-							placeholder="Search Tickets"
-							type="text"
-						></input>
-
-						<TicketGrid tickets={tickets} />
-
-						<div className="my-3">
-							<ClayPaginationBarWithBasicItems
-								active={page}
-								activeDelta={pageSize}
-								ellipsisBuffer={3}
-								ellipsisProps={{
-									'aria-label': 'More',
-									'title': 'More',
-								}}
-								onActiveChange={setPage}
-								onDeltaChange={setPageSize}
-								spritemap={Liferay.Icons.spritemap}
-								totalItems={totalCount}
-							/>
-						</div>
+							onActiveChange={setPage}
+							onDeltaChange={setPageSize}
+							spritemap={Liferay.Icons.spritemap}
+							totalItems={totalCount}
+						/>
 					</div>
+				</div>
 
-					<nav className="col-md-2 ml-auto">
-						<h6 className="text-uppercase">Filters</h6>
-						<ul>
-							{filters.map((_filter, index) => {
-								const isFilterSelected = compareFilters(
-									filter,
-									_filter
-								);
-
-								return (
-									<li key={index}>
-										<a
-											className={
+				<nav className="col-md-2 ml-auto">
+					<h6 className="text-uppercase">Filters</h6>
+					<ul>
+						{filters.map((_filter, index) => {
+							const isFilterSelected = compareFilters(
+								filter,
+								_filter
```

Comment:
Instead of naming the filter in the map `_filter`, rename the currently applied filter to `currentFilter`. This will make it much easier to understand what's going on.

Reply:
Also the compareFilters funciton doesn't seem necessary, you can just check if `currentFilter.value === filter.value`.